notebooks/official/training/pytorch_gcs_data_training.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<p>Vertex -&gt; Vertex AI</p><p> </p><p>Although -&gt; While</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>s/Vertex/"Vertex AI": Done.</p><p>s/Although/While: Ack</p>

Reply:
<p>"Although" is perfectly reasonable per word list: https://developers.google.com/style/word-list#letter-a</p>